sport/app/rugby/views/fragments/matchSummary.scala.html

```diff
@@ -27,9 +28,29 @@
     }
 }
 
-
+@status() = {
+    @theMatch.status match {
+        case Result      => { <abbr class="m-status">FT</abbr> }
```

Comment:
Perhaps add the `title` attribute to define the full description of the abbreviation. Not sure if it's valid without.